src/ast/passes/codegen_llvm.cpp

```diff
@@ -4870,6 +4897,57 @@ Pass CreateCompilePass(
                       });
 }
 
+Pass CreateLinkBitcodePass()
+{
+  return Pass::create("LinkBitcode",
+                      [](Imports &imports,
+                         CompiledModule &cm,
+                         CompileContext &ctx) -> Result<> {
+                        for (const auto &[_, bc] : imports.bitcode) {
+                          // Parse the underlying bitcode.
+                          auto buf = llvm::MemoryBuffer::getMemBuffer(
+                              StringRef(bc.data));
+                          assert(buf);
+                          auto mod = llvm::parseBitcodeFile(*buf, *ctx.context);
+                          if (!mod) {
+                            return mod.takeError();
+                          }
+
+                          // Modify to ensure everything is inlined. Note that
```

Comment:
Nit: I know I have the reputation of being anti-comment but might be good to also explain why the other removed/added attributes are important.

Reply:
Added.